### PR TITLE
en: snapshot-volume backup and restore

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:  https://github.com/pingcap/docs-tidb-operator/pull/2108
- Other reference link(s):

### CODE REVIEW

File: en/backup-restore-snapshot-perf.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+### Backup time consumption
+
+EBS snapshot backup using volumes consists of the following processes: creates a backup task, stops scheduling, stops GC, and obtains the backupts and volume snapshots. For more detailed information about these processes, see [Architecture of EBS snapshot volume backup and restore](volume-snapshot-backup-restore.md). Among these processes, creating a volume snapshot consumes most of the time. Volume snapshots are created in parallel, and the time taken to complete the entire backup task depends on when the most time-consuming volume is created.
+
+### Time consumption ratio of backup
+
+| Backup stage     | Time taken    | Total ratio | Remarks                                     |
+| :--------: | :---------: | :------: | :-------------------------------------: |
+| Create volume snapshots  | 16 m (50GB) | %99      | Including the time for creating AWS EBS snapshots                  |
+| Others        | 1s          | %1       | Including the time for stopping scheduling, disabling GC, and obtaining the backupts |
```
Comment (ran-huang) [inline en/backup-restore-snapshot-perf.md:23]:
```suggestion
| Create volume snapshots  | 16 m (50 GB) | 99%      | Including the time for creating AWS EBS snapshots                  |
| Others        | 1 s          | 1%       | Including the time for stopping scheduling, disabling GC, and obtaining the backupts |
```

File: en/backup-restore-snapshot-perf.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+2. Restore volumes.
+
+    TiDB Operator creates volume restore subtasks using BR. BR restores the data volumes from the snapshots to start TiKV.
+
+3. Start TiKV.
+
+    TiDB Operator mounts the TiKV volumes and starts TiKV.
+
+4. Restores data.
```
Comment (ran-huang) [inline en/backup-restore-snapshot-perf.md:71]:
```suggestion
4. Restore data.
```

File: en/backup-restore-snapshot-perf.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+5. Start TiDB.
+
+    TiDB is started and the restore is completed.
+
+### Time consumption ratio of restore
+
+| Restore stage     | Appropriate time taken | Restore ratio | Remarks                                                            |
+| :--------: | :---------: | :------: | :-------------------------------------------------------------: |
+| Creates clusters     | 30 s         | %1       | Including the time for downloading docker image and starting PD                                   |
+| Restores volumes     | 20 s         | %1       | Including the time for starting the BR Pod and restoring volumes                                         |
```
Comment (ran-huang) [inline en/backup-restore-snapshot-perf.md:84]:
```suggestion
| Creates clusters     | 30 s         |  1%      | Including the time for downloading docker image and starting PD                                   |
| Restores volumes     | 20 s         |  1%     | Including the time for starting the BR Pod and restoring volumes                                         |
```

File: en/backup-restore-snapshot-perf.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+| Starts TiKV   | 2～30 min    | 58%      | If there are ongoing transactions during backup, rocksDB consumes extra 5-30 minutes to start during restore |
```
Comment (ran-huang) [inline en/backup-restore-snapshot-perf.md:85]: 跟中文有点对不上

| 启动 TiKV | 2～30 min | 58% | TiKV 启动受卷性能影响，一般3分钟左右，当遇到有事务的备份做恢复时，此阶段耗时可能达到 30 分钟 |

File: en/backup-restore-snapshot-perf.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+| Restores data | 2～20 min    | 38%       |   Including the time for restoring data in the raft consensus layer and deleting MVCC data                    |
+| Starts TiDB   | 1 min        | 2%       |  Including the time for download the TiDB docker image and starting TiDB                     |
```
Comment (ran-huang) [inline en/backup-restore-snapshot-perf.md:87]:
```suggestion
| Starts TiDB   | 1 min        | 2%       |  Including the time for downloading the TiDB docker image and starting TiDB                     |
```

File: en/backup-restore-snapshot-perf.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+| :------: | :-----: | :---------------: | :--------: |
+| 50 GB    | 500 GB  | 7000IOPS/400MiB/s | 16 min    |
+| 100 GB   | 500 GB  | 7000IOPS/400MiB/s | 18 min    |
+| 200 GB   | 500 GB  | 7000IOPS/400MiB/s | 21 min   |
+| 500 GB   | 1024 GB | 7000IOPS/400MiB/s | 25 min   |
+| 1024 GB  | 3500 GB | 7000IOPS/400MiB/s | 34 min   |
+
+> **Note:**
+>
+> The preceding performance data is for reference only. The actual performance may vary.
```
Comment (ran-huang) [inline en/backup-restore-snapshot-perf.md:109]: 这个提示要不要放到文档开头？

File: en/backup-restore-snapshot-perf.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+summary: Learn about the performance of EBS snapshot backup and restore.
+---
+
+# Performance of EBS Snapshot Backup and Restore
+
+This document describes the performance of EBS snapshot backup and restore, the factors that affect performance, and the performance test results. The performance metrics are based on the AWS region `us-west-2`.
+
+## Backup performance
+
+This section introduces the performance of EBS snapshot backup using volumes, the factors that affect performance, and the performance test results. Backup performance is scalable and increases linearly with the number of TiKV nodes.
```
Comment (shichun-0415) [inline en/backup-restore-snapshot-perf.md:12]:
```suggestion
This section introduces the performance of EBS snapshot backup using volumes, the factors that affect performance, and the performance test results.
```

File: en/backup-restore-snapshot-perf.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+
+### Backup time consumption
+
+EBS snapshot backup using volumes consists of the following processes: creates a backup task, stops scheduling, stops GC, and obtains the backupts and volume snapshots. For more detailed information about these processes, see [Architecture of EBS snapshot volume backup and restore](volume-snapshot-backup-restore.md). Among these processes, creating a volume snapshot consumes most of the time. Volume snapshots are created in parallel, and the time taken to complete the entire backup task depends on when the most time-consuming volume is created.
```
Comment (shichun-0415) [inline en/backup-restore-snapshot-perf.md:16]:
```suggestion
EBS snapshot backup using volumes consists of the following processes: creates a backup task, stops scheduling, disables GC, and obtains the backupts and volume snapshots. For more detailed information about these processes, see [Architecture of EBS snapshot volume backup and restore](volume-snapshot-backup-restore.md). Among these processes, creating a volume snapshot consumes most of the time. Volume snapshots are created in parallel, and the time taken to complete the entire backup task depends on when the most time-consuming volume is created.
```

**Review by ran-huang — APPROVED** [review]

**Review by fengou1 — APPROVED** [review]

**Pull request merged**